airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts

```diff
@@ -121,6 +127,123 @@ describe("Connection main actions", () => {
     deleteDestination(destName);
   });
 
+  it("creates a connection, then edits the schedule type", () => {
+    const sourceName = appendRandomString("Test connection source cypress PokeAPI");
+    const destName = appendRandomString("Test connection destination cypress");
+
+    createTestConnection(sourceName, destName);
+
+    cy.get("div").contains(sourceName).should("exist");
+    cy.get("div").contains(destName).should("exist");
+
+    openSourceDestinationFromGrid(sourceName);
+
+    goToReplicationTab();
+
+    selectSchedule("Cron");
+    submitButtonClick();
+    checkSuccessResult();
+
+    selectSchedule("Manual");
+    submitButtonClick();
+    checkSuccessResult();
+
+    selectSchedule("Every hour");
+    submitButtonClick();
+    checkSuccessResult();
+
+    deleteSource(sourceName);
+    deleteDestination(destName);
+  });
+
+  // Analytics calls are not made in CI.
+  it.skip("Should make analytics calls", () => {
+    const sourceName = appendRandomString("Test connection source cypress PokeAPI");
+    const destName = appendRandomString("Test connection destination cypress");
+
+    createTestConnection(sourceName, destName);
+
+    cy.get("div").contains(sourceName).should("exist");
+    cy.get("div").contains(destName).should("exist");
+
+    openSourceDestinationFromGrid(sourceName);
+
+    goToReplicationTab();
+
+    cy.intercept("https://api.segment.io/v1/t").as("analyticsCall");
+
+    let analyticsCalls = 0;
+
+    selectSchedule("Cron");
+    cy.wait("@analyticsCall").then(() => {
+      analyticsCalls++;
+    });
+    submitButtonClick();
+    checkSuccessResult();
+
+    selectSchedule("Manual");
+    cy.wait("@analyticsCall").then(() => {
+      analyticsCalls++;
+    });
+    submitButtonClick();
+    checkSuccessResult();
+
+    selectSchedule("Every hour");
+    cy.wait("@analyticsCall").then(() => {
+      analyticsCalls++;
+      expect(analyticsCalls).to.eq(3);
+    });
+    submitButtonClick();
+    checkSuccessResult();
+
+    deleteSource(sourceName);
+    deleteDestination(destName);
+  });
+
+  it("Saving a connection's schedule type only changes expected values", () => {
+    cy.intercept("/api/v1/web_backend/connections/update").as("updateConnection");
+    cy.intercept("/api/v1/web_backend/connections/get").as("getConnection");
+
+    const sourceName = appendRandomString("Test update connection PokeAPI source cypress");
+    const destName = appendRandomString("Test update connection Local JSON destination cypress");
+
+    createTestConnection(sourceName, destName);
+
+    goToSourcePage();
+    openSourceDestinationFromGrid(sourceName);
+    openSourceDestinationFromGrid(`${sourceName} <> ${destName}`);
+
+    let loadedConnection: any = null; // Should be a WebBackendConnectionRead
```

Comment:
Just curious, do we have any plans to pull in types from the `airbyte-webapp` module in the future?

Reply:
I'm not sure what it would entail to do so. It might be trivial but no, I don't think we have a plan to do so. I wouldn't mind though.